web-ui/src/components/onboarding_sidebar/RightSidebar.css

```diff
@@ -0,0 +1,7 @@
+.wrapPanel {
+  width: 100%;
+}
+
+.sidebar__item{
+  display:flex;
```

Comment:
This file has not been formatted. This should be `display: flex;` with a space.

Reply:
I have reformatted this